<launch>
    <node name="pid_path_follower_node" pkg="pid_path_follower" type="pid_path_follower_node" output="screen">
        <param name="kp_heading" value="1.0" />
        <param name="ki_heading" value="0.0" />
        <param name="kd_heading" value="0.0" />
        <param name="kp_dist" value="0.5" />
        <param name="ki_dist" value="0.01" />
        <param name="kd_dist" value="0.1" />
        <param name="max_linear_speed" value="0.1" />
        <param name="max_angular_speed" value="0.22" />
        <param name="goal_tolerance" value="0.2" />
    </node>
</launch>
